rllib/core/testing/utils.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Tuple
+import gym
+import tensorflow as tf
+import torch
+import numpy as np
+
+
+from ray.rllib.core.rl_module.rl_module import RLModule
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.numpy import convert_to_numpy
+
+
+def do_rollouts(
+    env: gym.Env,
+    module_for_inference: RLModule,
+    num_rollouts: int,
+    framework: str = "tf",
+) -> Tuple[float, List[float], List[SampleBatch]]:
+    """Do rollouts of moduke_for_inference in env.
+
+    Args:
+        env: The environment to rollout in.
+        module_for_inference: The module to use for inference.
+        num_rollouts: The number of rollouts to do.
+        framework: The framework to use for inference.
+
+    Returns:
+        The mean return across num_rollouts, the return at each timestep, and
+            the rollouts.
+    """
+    _returns = []
```

Comment:
can I ask again why have all these prefix _?

Reply:
oh now there is no good reason. I'll go ahead and remove them.